src/main/java/org/candlepin/subscriptions/security/ApiSecurityConfiguration.java

```diff
@@ -161,7 +165,8 @@ protected void configure(HttpSecurity http) throws Exception {
         .anonymous() // Creates an anonymous user if no header is present at all. Prevents NPEs
         .and()
         .authorizeRequests()
-        .antMatchers("/**/openapi.*", "/**/version", "/api-docs/**", "/webjars/**")
+        .antMatchers(
+            "/**/openapi.*", "/**/internalopenapi.*", "/**/version", "/api-docs/**", "/webjars/**")
```

Comment:
Thoughts on making a constant at the top of this class named something like "URLS_PERMITTED_WITHOUT_AUTH"?

cc: @kahowell

Reply:
I think that would be cleaner, I'll update.